src/shared/views/topMenu/TopMenuHeader/TopMenuHeader.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import styles from './topMenuHeader.css'
+
+const TopMenuHeader = (props) => {
+  const titleTransform = (location) => {
+    const title = location.split('/')
+    if (title[0] === '') return 'Home'
```

Comment:
I'd prefer to always wrap `if()` in `{}` unless there is a specific reason not to.

For clarity, it makes it very easy to skim and realise a return will only occur if something else happens.